content/pages/download.md

```diff
@@ -2,11 +2,10 @@
 title: Download
 template: pages/download
 versions:
-  unstable:
-    name: 2.3.0 Beta
-    next_stable_version: 2.3.0
-    release_announcement: /news/2020-06-07-mixxx-2-3-beta-released/
-    download_manifest: https://downloads.mixxx.org/snapshots/2.3/manifest.json
+  stable:
+    name: 2.3.0
+    #release_announcement: /news/XXXX-XX-XX-mixxx-2-3-0-released/
```

Comment:
don't forget about adding this before merge.

Reply:
This need to be added after merge, because only after merge the URL is known. But it's just the backlink to the release announcement, so even if it's missing it's no big deal.